src/components/value-renderers/Header.tsx

```diff
@@ -1,13 +1,12 @@
-import { Text } from './Text';
-import { TitleProps } from './Title';
+import { Title, TitleProps } from './Title';
 
 interface HeaderProps extends TitleProps {
   sorted?: 'asc' | 'desc' | false;
 }
 export function Header({ sorted = false, ...other }: HeaderProps) {
   return (
     <div>
```

Comment:
Maybe this should be a `<th>` ?
Also I'm not sure `value-renderers` is the best place to put table-specific components.

Reply:
`<td/>` is added in Row component

Reply:
But this is a header, so we should have `<th>` in the DOM, not `<td>`

Reply:
> Also I'm not sure `value-renderers` is the best place to put table-specific components.

could you please suggest the right place?